MediaBrowser.Api/Playback/Hls/HLSCodecStringFactory.cs

```diff
@@ -0,0 +1,121 @@
+using System;
+using System.Text;
+using System.Linq;
+
+
+namespace MediaBrowser.Api.Playback
+{
+    /// <summary>
+    /// Get various codec strings for use in HLS playlists
+    /// </summary>
+    static class HLSCodecStringFactory
+    {
+
+        /// <summary>
+        /// Gets a mp3 codec string
+        /// </summary>
+        /// <returns>MP3 codec string</returns>
+        public static string GetMp3String()
+        {
+            return "mp4a.40.34";
+        }
+
+        /// <summary>
+        /// Gets an AAC codec string
+        /// </summary>
+        /// <param name="profile">AAC profile</param>
+        /// <returns>AAC codec string</returns>
+        public static string GetAACString(string profile)
+        {
+            StringBuilder result = new StringBuilder(10);
+            result.Append("mp4a");
+            if (string.Equals(profile, "HE", StringComparison.InvariantCultureIgnoreCase))
+            {
+                result.Append(".40.5");
+            } else
+            {
+                // Default to LC if profile is invalid
+                result.Append(".40.2");
+            }
+
+            return result.ToString();
+        }
+
+        /// <summary>
+        /// Gets a H.264 codec string
+        /// </summary>
+        /// <param name="profile">H.264 profile</param>
+        /// <param name="level">H.264 level</param>
+        /// <returns>H.264 string</returns>
+        public static string GetH264String(string profile, int level)
+        {
+            StringBuilder result = new StringBuilder(12);
+            string levelHex = level.ToString("X2");
+            result.Append("avc1");
+
+            if (string.Equals(profile, "baseline", StringComparison.InvariantCultureIgnoreCase))
+            {
+                result.Append(".42E0");
+            } else if (string.Equals(profile, "main", StringComparison.InvariantCultureIgnoreCase))
+            {
+                result.Append(".4D40");
+            } else if (string.Equals(profile, "high", StringComparison.InvariantCultureIgnoreCase))
+            {
+                result.Append(".6400");
+            } else
```

Comment:
```suggestion
            if (string.Equals(profile, "baseline", StringComparison.InvariantCultureIgnoreCase))
            {
                result.Append(".42E0");
            }
            else if (string.Equals(profile, "main", StringComparison.InvariantCultureIgnoreCase))
            {
                result.Append(".4D40");
            }
            else if (string.Equals(profile, "high", StringComparison.InvariantCultureIgnoreCase))
            {
                result.Append(".6400");
            }
            else
```

Reply:
Why is this `InvariantCultureIgnoreCase` instead of `OrdinalIgnoreCase`?

Reply:
I have no good reason, I'll fix that